core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -276,13 +279,17 @@ class SparkContext(config: SparkConf) extends Logging {
 
   private[spark] def ui: Option[SparkUI] = _ui
 
+  /**
+   * @return `Option` containing url to web ui.
```

Comment:
url -> URL and ui -> UI, but, I don't this comment adds much to the reader beyond what the signature says already. I'd improve or remove it.